src/app/component/comments/comments.module.ts

```diff
@@ -1,21 +1,20 @@
 import { NgModule } from '@angular/core';
-import {CommentsComponent} from './comments/comments.component';
-import {SharedModule} from '../shared/shared.module';
-import {CommonModule} from '@angular/common';
-import {CommentsRoutingModule} from './comments-routing.module';
-
+import { CommentsComponent } from './comments/comments.component';
+import { SharedModule } from '../shared/shared.module';
+import { CommonModule } from '@angular/common';
```

Comment:
Provide some convenient order of imports (eg. - angular modules at first place, next - third party and yours)

Reply:
fixed